storage/storage.go

```diff
@@ -53,12 +57,13 @@ func FindByName(name string) (*Tunnel, error) {
 	tun := store.Tunnels[name]
 
 	if tun == nil {
-		return nil, fmt.Errorf("alias could not be found: %s", name)
+		return nil, fmt.Errorf("tunnel could not be found: %s", name)
```

Comment:
This error message will be returned back to the user and using `tunnel` instead of `alias` might confuse them since tunnel usually refers to the communication protocol instead of the configuration associated to it (the alias).

How about keeping `alias`?

Reply:
Reverted